python/CMakeLists.txt

```diff
@@ -97,10 +97,12 @@ if(APPLE)
   if(NOT INSTALL_NAME_TOOL_EXECUTABLE)
     message(FATAL_ERROR "install_name_tool not found, please check.\n")
   endif()
-else(APPLE)
+endif()
+if(UNIX AND NOT APPLE)
```

Comment:
add
```
if(UNIX AND NOT APPLE)
        set(LINUX TRUE)
    endif()
```
in `configure.cmake`? so can use something like `if (LINUX) elseif (WIN32) elseif(APPLE)`?

Reply:
done.